scipy/spatial/ckdtree.pyx

```diff
@@ -706,7 +706,7 @@ cdef class cKDTree:
             When k == 1, the last dimension of the output is squeezed.
             Missing neighbors are indicated with infinite distances.
         i : ndarray of ints
-            The locations of the neighbors in ``self.data``.
+            The index locations of the neighbors in ``self.data``.
```

Comment:
I think you can drop the word "locations". Also, can you update the [KDTree.query](https://github.com/scipy/scipy/blob/2a1dc2ae8d2c944209f3053d1285dbe5edaff6cc/scipy/spatial/kdtree.py#L388) documentation to match.

Reply:
Done, I assumed "index" is better than "indexes" or "indices".
Also updated latest upstream commits. This PR should be good for merge.

Reply:
>I assumed "index" is better than "indexes" or "indices".

Grammatically, I think it should really be "The index of each neighbor" or "The indices of the neighbours".